### PR TITLE
Set request byte thresold

### PR DESCRIPTION
**Pull request by gauravphoenix**
fixes https://github.com/micronaut-projects/micronaut-gcp/issues/567

### CODE REVIEW

File: gcp-pubsub/src/main/java/io/micronaut/gcp/pubsub/support/DefaultPublisherFactory.java [hunk 1]
```diff
@@ -87,8 +87,7 @@ public Publisher createPublisher(@NonNull PublisherFactoryConfig config) {
                         .setDelayThreshold(batchSettings.getDelayThreshold())
                         .setElementCountThreshold(batchSettings.getElementCountThreshold())
                         .setIsEnabled(batchSettings.getIsEnabled())
-                        .setElementCountThreshold(batchSettings.getElementCountThreshold())
-                        .setDelayThreshold(batchSettings.getDelayThreshold())
```
Comment (graemerocher) [inline gcp-pubsub/src/main/java/io/micronaut/gcp/pubsub/support/DefaultPublisherFactory.java:91]: Why were these deleted?

Comment (gauravphoenix) [reply gcp-pubsub/src/main/java/io/micronaut/gcp/pubsub/support/DefaultPublisherFactory.java:91]: they have already been set

**Comment by gauravphoenix** [comment]
the build failure is unrelated to this PR
